content/en/dashboards/guide/using_vega_lite_in_wildcard_widgets.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Using Vega-Lite with Wildcard Widgets in Datadog
+further_reading:
+- link: "https://docs.datadoghq.com/dashboards/widgets/wildcard/"
+  tag: "Documentation"
+  text: "Learn more about the Wildcard widget"
+- link: "https://docs.datadoghq.com/dashboards/guide/widget_colors/"
+  tag: "Documentation"
+  text: "Selecting the right colors for your graphs"
+- link: "https://docs.datadoghq.com/dashboards/guide/context-links/#context-links-variables"
+  tag: "Documentation"
+  text: "Using Context Links in Dashboards"
+---
+
+## Overview
+
+When using Vega-Lite with Wildcard widgets in Datadog, you'll find extensions to the Vega-Lite specification which are unique to Datadog. This guide outlines the necessary configurations and considerations for effectively using Vega-Lite for data visualization in Datadog, ensuring compatibility with its unique specifications. By understanding and leveraging these specifications, you can create visually appealing and interactive data visualizations that are both effective and responsive to your thematic preferences.
+
+**Note**: Some extensions in Vega-Lite are exclusive to Datadog and might not function in the same way if exported to other tools that have Vega-lite.
+
+## Customizing the theming and Color Palettes
+
+Datadog provides a range of theming and color palette options to enhance the visual appeal of widgets. You can specify custom colors so that they blend in with the styling choices used by native Datadog widgets. If you set custom colors, the graph will not adjust colors when the app theme changes. By default, Datadog graphs adjust colors for text and axis marks to ensure readable contrast when viewed in dark mode. It's best to avoid setting custom colors for graph axes.
+
+Customized color, font, spacing, and other design settings are available. These settings apply automatically when using the theme switcher (`CTRL + OPT + D`).
+
+### Custom color palette
+
+While you can create custom color palettes using hex codes, using the Datadog color palette ensures automated switching between light and dark modes.
+
+Datadog offers additional color palettes beyond the public Vega color schemes, including:  
+- `dog_classic_area`  
+- `datadog16`  
+- `hostmap_blues`
+
+{{< whatsnext desc="Additional resources:" >}}
+    {{< nextlink href="/dashboards/guide/widget_colors/" >}}Learn more about Datadog color schemes and themes{{< /nextlink >}}
+    {{< nextlink href="https://docs.datadoghq.com/dashboards/guide/widget_colors/" >}}See Vega color schemes{{< /nextlink >}}
```

Comment:
```suggestion
    {{< nextlink href="/dashboards/guide/widget_colors/" >}}See Vega color schemes{{< /nextlink >}}
```
are both these links supposed to be the same?

Reply:
Nope, great catch! I updated your suggestion to reflect the correct link.